Library/Formula/pius.rb

```diff
@@ -10,7 +11,7 @@ def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
     # Replace hardcoded gpg path (WONTFIX):
     # https://sourceforge.net/p/pgpius/bugs/12/
-    inreplace "libpius/constants.py", "/usr/bin/gpg", "#{HOMEBREW_PREFIX}/bin/gpg"
+    inreplace "libpius/constants.py", %r{/usr/bin/gpg2?}, "#{HOMEBREW_PREFIX}/bin/gpg"
```

Comment:
That'd be great, thanks. It looks more or less fine if we get the comment added, but thank you for the heads up on being away - Helps us not worry a PR has gone dead if you stop responding for a while.